### ISSUE TITLE
How does this work exactly?

### ISSUE THREAD

**Issue by Lebedev21** [SrMotion/GTPS3#1]
"Also When u connected to server vyte will be downloading a RAT(Remote Administrator Tool) and getting access to your PC, BEWARE DO NOT PLAY"

How does that work? How can you get ratted just by connecting to a private server?
Any explanation, I'm curious?

**Comment by playingoDEERUX** [comment]
Yes, it downloads .exe from items.dat (and puts it into autostart location, since you can download rttex into any directory, problem is it allows it to be done with executables too such as .com and .exe file)

And I never even expected to see this going public some time...

I didn't expect even Vyte having this, let alone few other guys... Either way the fact it's leaked now is bad, though it isn't the worst thing that I'm aware of that can be done. It will definitely scare players on windows from playing GTPS though...

**Comment by Lebedev21** [comment]
Hmm, interesting.
This is certainly a new method.
Thanks for the information.

**Comment by SrMotion** [comment]
lets pin this issue

**Comment by SrMotion** [comment]
> Yes, it downloads .exe from items.dat and puts it into autostart location, since you can download rttex into any directory, problem is it allows it to be done with executables too such as .com and .exe file.
> 
> That means next time you start your PC, the RAT will be executed.
> 
> And I never even expected to see this going public some time...
> 
> I didn't expect even Vyte having this, let alone few other guys... Either way the fact it's leaked now is bad, though it isn't the worst thing that I'm aware of that can be done. It will definitely scare players on windows from playing GTPS though...
> 
> Good that I'm out of GT community for good now, lol.

thanks for the explanation

**Comment by playingoDEERUX** [comment]
@Lebedev21 It's not a new method really, it's just smth I personally never expected anyone to get the idea of this...

Also this isn't the worst thing that happened because Android/iOS users can still play safely, there's a worse thing that hopefully won't get leaked this way.

No problem for the explanation anyways, I feel like this is something the community should know about.

**Comment by SrMotion** [comment]
i will terminate him every channels now lol time to start a war

**Comment by pessiuff** [comment]
you can also run kernel mode drivers :)

**Comment by growtoups** [comment]
So basically the only way to play on pc without being ratted is to check the autostart folder before closing/ restarting your computer to delete the .exe
